bundles/FileExplorerBundle/src/Installer.php

```diff
@@ -10,19 +10,19 @@ class Installer extends SettingsStoreAwareInstaller
         'fileexplorer'
     ];
 
-    public function install()
+    public function install(): void
     {
         $this->addPermissions();
```

Comment:
Can you please also give the permissions a category like here? https://github.com/pimcore/pimcore/pull/14039/files#diff-2828c0270e1810c7d1d830a4a8598641b7d2cc717ac45500babdba3ef9a88d20R23